.github/workflows/publish-release.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           key: ${{ github.sha }}
       - name: Dry Run Publish
         # omit npm-token token to perform dry run publish
-        uses: MetaMask/action-npm-publish@v4
+        uses: MetaMask/action-npm-publish@v1
         with:
```

Comment:
These two variables aren't supported in v1, we can drop them here and re-add them when we update